Won't compile on OSX

```
FileSystemPosix.cpp:114:25: error: no member named 'st_mtim' in 'stat'
                if (localtime_r(&info.st_mtim.tv_sec, &timeparts) != nullptr) {
                                 ~~~~ ^
FileSystemPosix.cpp:118:34: error: no member named 'st_mtim' in 'stat'
                        mtime += Time::TimeDelta(info.st_mtim.tv_nsec / ...
                                                 ~~~~ ^
FileSystemPosix.cpp:199:8: error: use of undeclared identifier 'fstatat'
                        if (fstatat(dirfd(dir), entry->d_name, &info, 0) == 0) {
                            ^
3 errors generated.
```

--- a/src/posix/FileSystemPosix.cpp
+++ b/src/posix/FileSystemPosix.cpp
@@ -115,9 +115,6 @@ namespace FileSystem {
 			mtime = Time::DateTime(
 				1900 + timeparts.tm_year, timeparts.tm_mon+1, timeparts.tm_mday,
 				timeparts.tm_hour, timeparts.tm_min, timeparts.tm_sec);
-#ifndef __APPLE__
-			mtime += Time::TimeDelta(info.st_mtim.tv_nsec / 1000, Time::Microsecond);
-#endif
 		}
 
 		return ty;

